pkgs/development/python-modules/indexed_gzip/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, cython
+, pytest-runner
+, zlib
+, numpy
+}:
+
+buildPythonPackage rec {
+  pname = "indexed_gzip";
+  version = "1.8.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "87d26062af4ac66927691b8c82afb961603cb5468593e95c964a9d3409ebfdc2";
+  };
+
+  nativeBuildInputs = [ cython ];
+
+  buildInputs = [ zlib ];
+
+  checkInputs = [ pytest-runner numpy ];
```

nibabel is a weird one, but looks like it handles some subset of neuro-imaging file formats
```suggestion
  checkInputs = [ pytest pytest-runner numpy nibabel ]; 
```

---

I have added the changes but I was unable to test them. After waiting 2h or so for LLVM and hundreds of other packages to build, it fails in the middle of compiling hundreds of rust packages because of insufficient disk space, where the "disk" seems to refer to my 6 GiB /run/user/1000 tmpfs mount.